lib/electrum_client.ex

```diff
@@ -15,6 +15,8 @@ defmodule ElectrumClient do
     Broadcast
   }
 
+  @callback get_transaction(txid :: String.t()) :: map()
```

Comment:
It would be more accurate to state that the function return is a `%BitcoinLib.Transaction{}`

Reply:
It returns something else https://github.com/RooSoft/electrum_client/blob/main/lib/calls/blockchain/transaction/get_transaction.ex#L139-L145 

```
%{
      block_hash: block_hash,
      time: DateTime.from_unix!(block_time),
      confirmations: confirmations,
      vsize: vsize,
      transaction: Transaction.parse(hex) |> elem(1)
    }
```

The `%BitcoinTransaction{}` is just a part of that "something". Initially, I created `ElectrumClient.Transaction` but later discarded it because I wasn't sure if it was the right thing.